debugpy crashing when breakpoints are set on Python 3.13

When trying to debug code in VS Code using cartopy (v0.25.0) with Python 3.13 (on Apple silicon), I'm getting the following exception on importing cartopy when breakpoints are set:

```
Exception has occurred: AssertionError
<code object _call_with_frames_removed at 0x104ea0570, file "<frozen importlib._bootstrap>", line 480> != <code object PyInit_trace at 0x1199aedb0, file "lib/cartopy/trace.pyx", line 1>
KeyError: (1, 'xx/envs/default/lib/python3.13/site-packages/lib/cartopy/trace.pyx', <code object PyInit_trace at 0x1199aedb0, file "lib/cartopy/trace.pyx", line 1>)

During handling of the above exception, another exception occurred:

  File "xx.py", line 1, in <module>
    import cartopy
AssertionError: <code object _call_with_frames_removed at 0x104ea0570, file "<frozen importlib._bootstrap>", line 480> != <code object PyInit_trace at 0x1199aedb0, file "lib/cartopy/trace.pyx", line 1>
```

Looking around, seems related to issues that have been reported in other packages using Cython:
#https://github.com/aio-libs/frozenlist/issues/658
#https://github.com/CoolProp/CoolProp/issues/2612

Not sure if this is something that can be fixed in cartopy? Thanks!

--- a/setup.py
+++ b/setup.py
@@ -45,7 +45,7 @@ if USE_CYTHON:
     # We need to explicitly cythonize the extension in order
     # to control the Cython compiler_directives.
     from Cython.Build import cythonize
-    compiler_directives = {"profile": CYTHON_COVERAGE, "linetrace": CYTHON_COVERAGE}
+    compiler_directives = {"profile": cython_coverage, "linetrace": cython_coverage}
     extensions = cythonize(extensions, compiler_directives=compiler_directives)
 
 

